.gitignore

```diff
@@ -2,6 +2,7 @@
 cache/
 out/
 artifacts/
+lib/
```

Comment:
typically we want to keep lib so ppl can just run `forge install` when playing with the repo